reccmp/isledecomp/cvdump/types.py

```diff
@@ -160,36 +160,36 @@ class CvdumpTypesParser:
 
     # LF_FIELDLIST class/struct member (1/2)
     LIST_RE = re.compile(
-        r"\s+list\[\d+\] = LF_MEMBER, (?P<scope>\w+), type = (?P<type>.*), offset = (?P<offset>\d+)"
+        r"\s+list\[\d+\] = LF_MEMBER, (?P<scope>\w+), type = (?P<type>[^,]*), offset = (?P<offset>\d+)"
```

Comment:
These are refactors purely for safety: We shouldn't have a `.*` or `.+` in a match group since it is very unstable regarding unknown syntax. I had a bug (not in this line) where it matched `"element_type": "0x2FD1, Containing class = 0x1165"`. Excluding commas seems safer and there should never be commas in these match groups